fakesnow/connection.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
```

Comment:
So i did that, but it introduced a circular import issue because transforms need to import the database/table name.

Thus I created a `global_database.py` instead, which leaves `connection.py` with a largely unaltered `FakeSnowflakeConnection`. I can just put that back in `fakes.py` if you want, or leave it here.